airflow/config_templates/config.yml

```
@@ -1080,6 +1080,13 @@
       type: string
       example: ~
       default: "30"
+    - name: site_title
+      description: |
+        Sets a title to show in the browser tab and the DAGs overview page
+      version_added: ~
+      type: string
+      example: ~
+      default:
```

We could actually add the default here, so that users can easily override it.

```suggestion
      default: Airflow
```

---

Thoughts here @pcandoalmeida ?

---

Hiya @kaxil sorry didn't see this! Yes, this defaults the page title to Airflow. By default, we have `Airflow - Airflow` unless there is a specific instance name like `Airflow - Dev`.